script/Arcueid/Day 2/02_00_ARC02_1_1.txt

```diff
@@ -206,6 +206,7 @@ Ehh, the girls would probably expose my age anyway so let's just say I'm 25 year
 [sha:209b1fc8017a4687108bbdaf3d4186c7d6b0899c]{
 -- Page 139, Offset 2965.
 -- 「教会？　なに先生、シスターなの!?　っていうか、趣味でシスターやってるんですか!?」
+// sister or nun?
```

Comment:
Personal opinion would be to localize Sister as Nun, but maybe run it by the group

Reply:
nun